### PR TITLE
BaSP-MR-268: [Not-Allowed] Add a new view for /auth-not-allowed

### PR DESCRIPTION
**Pull request by lorenzomauro59**
- Add the header to this component.
- Display something like the image(example)
- Add a button to go to the home, and the text below to go back

### CODE REVIEW

**Comment by lorenzomauro59** [comment]
[[Not-Allowed] Add a new view for /auth-not-allowed](https://trello.com/c/PfYdIwqQ/268-not-allowed-add-a-new-view-for-auth-not-allowed)

File: src/Components/Auth/NotAllowed/index.jsx [hunk 1]
```diff
@@ -23,21 +26,59 @@ const NotAllowed = () => {
...
-          <h2>
-            You are not logged in. You will be redirected to the Login form in {countdown}{' '}
-            seconds...
-          </h2>
-        </>
-      )}
-    </Container>
+    <>
+      <Header />
+      <Container isLogin={true}>
```
Comment (fernandofernandezuy) [inline src/Components/Auth/NotAllowed/index.jsx]: If you change the prop isLogin for center={true} the view wont scroll

File: src/Components/Auth/NotAllowed/index.jsx [hunk 2]
```diff
@@ -23,21 +25,59 @@ const NotAllowed = () => {
...
+          </>
+        ) : (
+          <>
+            <div className={styles.notAllowedContainer2}>
+              <img
+                src={`${process.env.PUBLIC_URL}/assets/images/Not-allowed/secureheight.png`}
+                alt="Secure-image"
+              />
+              <h3>
+                You are not logged in. You will be redirected to the Login form in {countdown}{' '}
```
Comment (AleMac17) [inline src/Components/Auth/NotAllowed/index.jsx]: "You are not logged in. You will be redirected to the Login in"

File: src/Components/Auth/NotAllowed/notAllowed.module.css [hunk 3]
```diff
@@ -1,5 +1,54 @@
...
+    max-width: 30vw;
+    max-height: 90vh;
+  }
+
+  .notAllowedContainer > img {
+    width: 15vw;
+    margin-right: 5vw;
+  }
+
+  .notAllowedContainer > Button {
```
Comment (AleMac17) [inline src/Components/Auth/NotAllowed/notAllowed.module.css]: Can you fix the ident?

**Review by EmaBaldor — APPROVED** [review]
Good job!

**Pull request merged**